hrm-domain/hrm-service/service-tests/contacts.test.ts

```diff
@@ -1649,5 +1650,32 @@ describe('/contacts route', () => {
         });
       });
     });
+    describe('DELETE', () => {
+      const subRoute = contactId => `${route}/${contactId}/connectToCase`;
+      test('should return 401', async () => {
+        const response = await request.delete(subRoute(existingContactId));
+
+        expect(response.status).toBe(401);
+        expect(response.body.error).toBe('Authorization failed');
+      });
+
+      test('should return 200', async () => {
+        const response = await request.delete(subRoute(existingContactId)).set(headers);
+
+        const contact = await getContactById(existingContactId, accountSid);
```

Comment:
There is a 'get contact by id' function in the HRM production code that you can use here instead of duplicating it in the test code